SaltStack docs (#51)
DataDog/datadog-formula: README.md

# Datadog Formula

The Datadog SaltStack formula is used to install the Datadog Agent and the Agent-based integrations (checks). For more details on SaltStack formulas, see the [Salt formulas installation and usage instructions][1].

## Setup

> ### Installation

Reviewer: I have recently updated our in-application docs to fix some mistakes in the installation and deployment steps, could you add them here too?

Author: @KSerrania - Are you talking about this PR?
https://github.com/DataDog/dogweb/pull/43489

If so, I updated based on that. Is there anything incorrect here?

Reviewer: I'm talking about https://github.com/DataDog/dogweb/pull/45800, which adds some corrections to the PR you mentioned.

```

#### Install settings

Under `install_settings`, configure the Agent installation option:

- `agent_version`: The version of the Agent to install (defaults to the latest Agent v7).

The example below installs Agent v6.14.1:

```text
  datadog:
    install_settings:
      agent_version: 6.14.1
```

#### Integrations

To add an Agent integration to your host, use the `checks` variable with the check's name as the key. Each check has two options:

| Option    | Description                                                                                                                                                             |
|-----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| `config`  | Add the configuration options to write to the check's configuration file:<br>Agent v6 & v7: `<confd_path>/<check>.d/conf.yaml`<br>Agent v5: `<confd_path>/<check>.yaml` |
| `version` | For Agent v6 & v7, the version of the check to install (defaults to the version bundled with the Agent).                                                                |

> Below is an example for the using v1.4.0 of the [Directory][3] integration monitoring the `/srv/pillar` directory:

Reviewer: ```suggestion
Below is an example to use v1.4.0 of the [Directory][3] integration monitoring the `/srv/pillar` directory:
```
